src/App.js

```diff
@@ -251,9 +252,12 @@ function App() {
         try {
             const fileName = params.row.name;
             const attachmentUrls = await fileManager.getAttachmentUrls(params.row.path);
-            const base64AttachmentUrls = btoa(JSON.stringify(attachmentUrls)).replace(/\+/g, '~').replace(/\//g, '_').replace(/=/g, '-');
+            const base64AttachmentUrlsBase = JSON.stringify(attachmentUrls);
+            const encodedUrls = pako.deflate(base64AttachmentUrlsBase.replace(/\?(.*?)"/g, '"'));
```

Comment:
Why the added `replace` call?

Reply:
The extra.replace calls trims the returned URL and removes the query parameters

They don't seem to do anything and the files resolved without them!

Reply:
Okay, I feel like it would be more readable to remove these query params in `DiscordFileStorage.getAttachmentUrls` for each URL using some URL parsing from JS instead of doing it with regex.

Reply:
Okay, I moved it to the getAttachmentUrls() function
```javascript
    async getAttachmentUrls(messageIds) {
        const attachmentUrls = [];
        for (let id of messageIds) {
            const message = await this.webhookClient.getMessage(id);
            const url = message.attachments[0].url;
            const urlObj = new URL(url);
            const shortenedUrl = `${urlObj.protocol}//${urlObj.host}${urlObj.pathname}`;
            attachmentUrls.push(shortenedUrl);
        }
        console.log(attachmentUrls);
        return attachmentUrls;
    }
```